### PR TITLE
Update categoryTranslation.js

### PR DESCRIPTION
**Pull request by fredao**

### CODE REVIEW

File: data/categoryTranslation.js [hunk 1]
```diff
@@ -181,8 +197,9 @@ module.exports = {
           "DE": "Kennst Du schon …",
           "ES": "Sabías que …",
           "FR": "Connaissez-vous …",
-          "CZ": "Did you know …",
+          "CZ": " Did you know … ",
```
Comment (alexandre-mbm) [inline data/categoryTranslation.js]: This line is better like before.

File: data/categoryTranslation.js [hunk 2]
```diff
@@ -6,7 +6,8 @@ module.exports = {
           "ES": "-- no category yet --",
           "FR": "-- no category yet --",
           "CZ": "-- no category yet --",
-          "PT": "-- ainda sem categoria --",
+          "PT": "-- no category yet --",
```
Comment (alexandre-mbm) [inline data/categoryTranslation.js]: This line is better like before.

**Comment by fredao** [comment]
why all checks have failed? can I see why?

**Comment by alexandre-mbm** [comment]
From https://travis-ci.org/TheFive/osmbc/builds/93116006 lines 381-392:
```
Start Travis Test With Coverage Upload
Reading Config from: /home/travis/build/TheFive/osmbc/config.test.json
Transformation error; return original code
{ [Error: Line 238: Unexpected string]
  index: 7288,
  lineNumber: 238,
  column: 11,
  description: 'Unexpected string' }

/home/travis/build/TheFive/osmbc/data/categoryTranslation.js:238
          "RU": "--не опубликовано",
          ^^^^
```

**Pull request merged**